### PR TITLE
Fix ballerina run failed to load sql drivers

### LINKED ISSUE #19554: Ballerina Module Fails to Load MySQL Driver Instance
**Description:**
When I try to run the Ballerina module named "issuer" located in 
[blockchain-iam-group-chat.zip](https://github.com/ballerina-platform/ballerina-lang/files/3765829/blockchain-iam-group-chat.zip) I get the following output,

```
miyurud@miyurud:~/git/blockchain-iam-group-chat$ ballerina run issuer
Compiling source
	wso2/issuer:0.1.0

Creating balos
	target/balo/issuer-2019r3-java8-0.1.0.balo

Generating executables
	target/bin/issuer.jar

Running executables

error: {ballerinax/java.jdbc}ApplicationError message=error in sql connector configuration: Failed to get driver instance for jdbcUrl=jdbc:mysql://192.168.32.1:3306/ssidb:No suitable driver
	at ballerinax.java_jdbc:createClient(jdbc_client.bal:87)
	   ballerinax.java_jdbc.Client:__init(client_endpoint.bal:27)
```

However, if I build this multi-module project using the following command,

`ballerina build -a --skip-tests`

and then run the module as follows, 

`ballerina run target/bin/issuer.jar`

I can start running this module successfully.

**Steps to reproduce:**
Download and extract the attached zip file ([blockchain-iam-group-chat.zip](https://github.com/ballerina-platform/ballerina-lang/files/3765829/blockchain-iam-group-chat.zip)) , change directory to "blockchain-iam-group-chat" folder, and issue ballerina build issuer. You may also have to change the path of the platform.libraries entry of the Ballerina.toml file to match with your environment. The corresponding "utils.jar" is located on the directory "utils-dir".

Next, issue the command,

`ballerina build -a --skip-tests`

Then run the module,

`ballerina run issuer`

**Affected Versions:**
Ballerina 1.0.2-SNAPSHOT

**OS, DB, other environment details and versions:**
Ubuntu 16.04

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by warunalakshitha**
## Purpose
$Subject

Fixes #19554

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: tests/jballerina-integration-test/src/test/resources/packaging/spi/TestProject/src/module/funcs.bal [hunk 1]
```diff
@@ -1,5 +1,11 @@
 import ballerina/io;
+import ballerinax/java;
 
 public function main() {
-    io:println("Hello World");
+    io:println(getDriversAsString());
 }
+
+public function  getDriversAsString() returns string = @java:Method {
+      class:"PrintDrivers"
```
Comment (Kishanthan) [inline tests/jballerina-integration-test/src/test/resources/packaging/spi/TestProject/src/module/funcs.bal]: Where is the PrintDrivers class loaded?

Comment (warunalakshitha) [reply tests/jballerina-integration-test/src/test/resources/packaging/spi/TestProject/src/module/funcs.bal]: its test jar i created jballerina-integration-test/src/test/resources/packaging/spi/TestProject/libs/drivers.jar

File: cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/model/ExecutableJar.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import java.util.HashSet;
+
+/**
+ * Class for holding contents of executable jar.
+ *
+ * @since 1.0.4
+ */
+public class ExecutableJar {
+
+    public Path moduleJar;
```
Comment (Kishanthan) [inline cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/model/ExecutableJar.java]: Can this cause NPE in any case?

Comment (warunalakshitha) [reply cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/model/ExecutableJar.java]: module jar is the one create with actual ballerina source. Only place we are using is to get the init class name for main. So it  never can be null.

**Pull request merged**